components/withBreadcrumbs.tsx

```diff
@@ -35,9 +35,10 @@ const WithBreadcrumbs: FC = () => {
     // Reduce the pathList to a breadcrumbs array by finding each path in the current navigation layer,
     // updating the currentNode to the found node's items(next layer) for the next iteration.
     return pathList.reduce((breadcrumbs, path) => {
-      const nodeWithCurrentPath = currentNode.find(
-        ([nodePath]) => nodePath === path
-      );
+      const nodeWithCurrentPath = currentNode.find(([nodePath]) => {
+        const matchs = path.toLowerCase().includes(nodePath.toLowerCase());
```

Comment:
Does a variable need to be assigned here? 🤔 can you just do

```tsx
return path.toLowerCase().includes(nodePath.toLowerCase());
```

?

Reply:
@ovflowd  sure will change that, I was debugging with logging the output so forgot to remove that afterward..